client/Dockerfile-client

```diff
@@ -0,0 +1,40 @@
+# Our First stage, that builds the application
+FROM node:12.13.0-alpine AS react-builder
+WORKDIR app
+
+# Copy all files 
+COPY . .
+
+# Install dependencies
+RUN yarn
+
+# set graphql server base url
+ARG REACT_APP_GRAPHQL_BASE_URL
+ENV REACT_APP_GRAPHQL_BASE_URL=$REACT_APP_GRAPHQL_BASE_URL
```

Comment:
Build arguments (`ARG`) are exported into environment by default. No need for an extra `ENV` statement.